src/azure-cli/azure/cli/command_modules/acr/tests/latest/test_split_registry_and_image.py

```diff
@@ -0,0 +1,58 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import unittest
+import importlib
+
+from knack.util import CLIError
+
+acr_import = importlib.import_module('azure.cli.command_modules.acr.import')
+
+
+class TestSplitRegistryAndImage(unittest.TestCase):
+
+    VALID_ACR_LOGIN_SERVER = 'myregistry.azurecr.io'
+    VALID_DOCKER_LOGIN_SERVER = 'docker.io'
+    VALID_IMAGE = 'myrepo/myimage:latest'
+
+    VALID_SOURCE_PATTERN = '{}/{}'
+    VALID_ACR_SOURCE = VALID_SOURCE_PATTERN.format(VALID_ACR_LOGIN_SERVER, VALID_IMAGE)
+    VALID_DOCKER_SOURCE = VALID_SOURCE_PATTERN.format(VALID_DOCKER_LOGIN_SERVER, VALID_IMAGE)
+
+    SINGLE_SLASH = '/'
+    SINGLE_DOT = '.'
+    EMPTY_SOURCE = ''
+
+    def test_split_valid_acr_source(self):
+        login_server, image = acr_import._split_registry_and_image(TestSplitRegistryAndImage.VALID_ACR_SOURCE)
+        self.assertEqual((login_server, image), (TestSplitRegistryAndImage.VALID_ACR_LOGIN_SERVER, TestSplitRegistryAndImage.VALID_IMAGE))
```

Comment:
Nice tests and nice to try to abstract these variables. It could be clear to define some of the inputs within each test. Also using self or defining constants at the module level, might be clearer. That being said, I see why you want to keep constants at the class level too to keep things modular.

Here are some suggestions...
```suggestion
    def test_split_valid_acr_source(self):
        input_acr_source = 'myregistry.azurecr.io/myrepo/myimage:latest'
        login_server, image = acr_import._split_registry_and_image(input_acr_source)
        self.assertEqual((login_server, image), (self.VALID_ACR_LOGIN_SERVER, self.VALID_IMAGE))
```